alibi_detect/cd/lsdd.py

```diff
@@ -18,7 +18,8 @@ def __init__(
             preprocess_x_ref: bool = True,
             update_x_ref: Optional[Dict[str, int]] = None,
             preprocess_fn: Optional[Callable] = None,
-            sigma: Optional[np.ndarray] = None,
+            # sigma: Optional[np.ndarray] = None,
+            # kernel: BaseKernel = None,
```

Comment:
Is the `kernel` here erroneous?

Reply:
yes, thanks for noticing, will remove all these redundant bits. (and probably use some unique character for these comment-outs next time for easy search...)